docs/pages/configuration/pipelines/functions/run_dependency_pipelines.mdx

```diff
@@ -2,3 +2,37 @@
 title: Function run_dependency_pipelines
 sidebar_label: run_dependency_pipelines
 ---
+
+Runs dependencies for pipelines.
```

Comment:
I think this needs more clarification. I know the file says it `deploys` the dependencies, but I'm not sure exactly how to describe this.

Reply:
@FabianKramm This is another great question. Can you answer this for us?

Reply:
Yep good question, in DevSpace dependencies are essentially other DevSpace projects that can be referenced from your current project. This function executes a pipeline within a dependency. So you are correct, `deploy` is probably the wrong term here as the dependency pipeline is rather executed than deployed as it could have a totally different functionality.